cvat-core/tests/api/annotations.cjs

```diff
@@ -87,6 +87,22 @@ describe('Feature: get annotations', () => {
         }
 
         expect(annotations[0].shapeType).toBe('skeleton');
+    });
+
+    test('handle page interrupt during annotation retrieval', async () => {
+        const job = (await cvat.jobs.get({ jobID: 101 }))[0];
+
+        // Mocking the behavior of job.annotations.get() to simulate a page interrupt
+        const mockAnnotationsGet = jest.spyOn(job.annotations, 'get');
+
+        // Reject the promise with an error to simulate a page interrupt
+        mockAnnotationsGet.mockRejectedValueOnce(new Error("Cannot read properties of null (reading 'annotations')"));
+
+        // Expectation: Retrieving annotations should throw an error
+        await expect(job.annotations.get(0)).rejects.toThrow("Cannot read properties of null (reading 'annotations')");
+
+        // Ensuring that job.annotations.get() was called with the correct arguments
+        expect(mockAnnotationsGet).toHaveBeenCalledWith(0);
```

Comment:
1. The code does not make sense, because the test tests itself
2. Perhaps we need Cypress E2E tests to cover the bugfix, not jest. Because issue on `cvat-ui` level. On `cvat-core` level we can't even reproduce the problem.

Reply:
The test was aiming to simulate a scenario where fetching annotations encounters an error. It was meant for verifying that the method correctly throws an error with the expected message when encountering a page interrupt.

If the issue is more related to UI behavior, then let me focus on E2E tests with Cypress.